### PR TITLE
Story/dfc 11327 sitefinity upgrade to live

### PR DESCRIPTION
**Pull request by vallabha24**

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests.csproj [hunk 1]
```diff
@@ -187,6 +187,7 @@
     <AdditionalFiles Include="..\stylecop.json">
       <Link>stylecop.json</Link>
     </AdditionalFiles>
+    <None Include="app.config" />
```
Comment (Gurnaik-Randhawa) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests.csproj]: Do we need this app.config

File: DFC.Digital/DFC.Digital.Web.Sitefinity.Logging/DFC.Digital.Web.Sitefinity.Logging.csproj [hunk 2]
```diff
@@ -239,6 +239,7 @@
     <AdditionalFiles Include="..\stylecop.json">
       <Link>stylecop.json</Link>
     </AdditionalFiles>
+    <None Include="app.config" />
```
Comment (Gurnaik-Randhawa) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.Logging/DFC.Digital.Web.Sitefinity.Logging.csproj]: Do we need this app.config?

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests.csproj (hunk 1 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests.csproj]: This was resolved now

File: DFC.Digital/DFC.Digital.Web.Sitefinity.Logging/DFC.Digital.Web.Sitefinity.Logging.csproj (hunk 2 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.Logging/DFC.Digital.Web.Sitefinity.Logging.csproj]: This was resolved now

**Comment by vallabha24** [comment]
This PR is not valid anymore as we created a new sitefinity upgrade from Master branch

**Pull request closed without merging**